pkg/virt-launcher/virtwrap/api/converter.go

```diff
@@ -609,13 +613,10 @@ func configPortForward(qemuArg *Arg, iface v1.Interface) error {
 			forwardPort.Protocol = DefaultProtocol
 		}
 
-		// Check if PodPort is configure If not Get the same Port as the vm port
-		if forwardPort.PodPort == 0 {
-			forwardPort.PodPort = forwardPort.Port
+		if _, ok := configuredPorts[fmt.Sprintf("%s-%d", forwardPort.Protocol, forwardPort.Port)]; !ok {
+			qemuArg.Value += fmt.Sprintf(",hostfwd=%s::%d-:%d", strings.ToLower(forwardPort.Protocol), forwardPort.Port, forwardPort.Port)
+			configuredPorts[fmt.Sprintf("%s-%d", forwardPort.Protocol, forwardPort.Port)] = struct{}{}
```

Comment:
nit: you could construct it once in line before the if and then reuse in both if and here.